src/AzureSqlResourceManager.ts

```diff
@@ -2,6 +2,9 @@ import { IAuthorizer } from 'azure-actions-webclient/Authorizer/IAuthorizer';
 import { WebRequest } from 'azure-actions-webclient/WebClient';
 import { ServiceClient as AzureRestClient, ToError, AzureError } from 'azure-actions-webclient/AzureRestClient'
 
+// API docs: https://docs.microsoft.com/rest/api/sql
+const SqlApiVersion = '2021-11-01';
```

Comment:
nit: do we have a convention for constants in this repo? e.g. SQL_API_VERSION?

Reply:
No convention I'm aware of, although we should probably have a conversation about that in the future as we implement more features!